### PR TITLE
Add project content features

### PR DESCRIPTION
**Pull request by BenMKT**
The following were added to my project:

-     HTML & CSS linters.
-     A descriptive README file.
-     index.html & styles.css files with relevant code context.
-     Added `test.md` file to `.gitignore` file and also node_modules.

### CODE REVIEW

File: README.md
Comment (Efiamotu-1) [inline README.md:1]: - your project's readme is curently missing a couple of important sections kindly read the description below on the best way to create your professional readme :+1: 
- * [ ]  For the project requirements, kindly ensure that your README file is descriptive and professional. And the required template to be used here is the **[Microverse README template](https://github.com/microverseinc/readme-template)**, so ensure that:
* All main headers from the [README template](https://github.com/microverseinc/readme-template) are present in a your repo.
* The only optional section is `FAQ`
* you delete the `live demo` section if the you decide not to deploy the application.
* Make sure the you have added the section `Future Features`, including 1-3 potential features to be added.
* you remove the section `description`, i.e.` "> Describe the tech stack and include only the relevant sections that apply to your project."` should be removed and filled in with their custom content.
* you remove template logo. Optionally, the you can replace it with your own logo you they have one.
* No broken link or image
- for further assistance see this [students repo for his descriptive readme](https://github.com/Mike111222/Hello-Microverse-Project/blob/main/README.md)

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @BenMKT , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,171 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
Comment (jmonto55) [inline README.md]: - [ ] Great work implementing your professional README file 💯. However, please kindly ensure to remove the Microverse logo from here🤓.

**Review by jmonto55 — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES 🟡

Hi @BenMKT 👋🏽, 

Great work on making the changes requested by a previous reviewer 👌
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

AlmostThereGIF



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdallahmalima — APPROVED** [review]
Hi @BenMKT , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter Errors✔
- Descriptive PR title and description✔
-  GitHub flow✔
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**